### PR TITLE
Dev_Phawat

### PR DESCRIPTION
**Pull request by plphxwxt**
This is my new feauture

### CODE REVIEW

File: lab-exercise-1/PhawatFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class PhawatFeature {
+    public void feature() {
+        // this is a new feature
+    }
```
Comment (nickttrps) [inline lab-exercise-1/PhawatFeature.java:4]: susu na puen

File: lab-exercise-1/PhawatFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class PhawatFeature {
+    public void feature() {
```
Comment (tnktian) [inline lab-exercise-1/PhawatFeature.java:2]: gogogogogoogo let's continue this project!

File: lab-exercise-1/PhawatFeature.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+        // this is a new feature
```
Comment (sRubyy) [inline lab-exercise-1/PhawatFeature.java:3]: Well done dev Phawat

**Comment by arthasem** [comment]
<html>
<body>


⊂(･ω･*⊂)
--



</body>
</html>

File: lab-exercise-1/PhawatFeature.java (hunk 2 above)
Comment (YunduoLuo) [inline lab-exercise-1/PhawatFeature.java:2]: nice work dude:)

**Review by sRubyy — COMMENTED** [review]
nice one let continue our project

**Review by YunduoLuo — COMMENTED** [review]
nice.

File: lab-exercise-1/PhawatFeature.java (hunk 2 above)
Comment (arthasem) [inline lab-exercise-1/PhawatFeature.java:2]: Good job bruh

File: lab-exercise-1/PhawatFeature.java (hunk 3 above)
Comment (arthasem) [reply lab-exercise-1/PhawatFeature.java:3]: Good job bruh

**Pull request merged**